agent/config/config_unix.go

```diff
@@ -80,6 +80,7 @@ func DefaultConfig() Config {
 		PollMetrics:                         false,
 		PollingMetricsWaitDuration:          DefaultPollingMetricsWaitDuration,
 		NvidiaRuntime:                       DefaultNvidiaRuntime,
+		AppMeshEnabled:                      false,
```

Comment:
Do you also need to update the test case **TestConfigDefault** in config_unix_test.go?

Reply:
updated